frontend/app/src/components/price-manager/latest/LatestPriceTable.vue

```diff
@@ -0,0 +1,230 @@
+<template>
+  <div>
+    <v-row justify="end" class="mb-10">
+      <v-col cols="auto">
+        <price-refresh :assets="assets" />
+      </v-col>
+    </v-row>
+    <card outlined-body>
+      <template #title>
+        <refresh-button
+          :loading="loading"
+          :tooltip="tc('price_table.refresh_tooltip')"
+          @refresh="refresh"
+        />
+        <div>
+          {{ tc('price_table.latest.title') }}
+        </div>
+      </template>
+      <slot />
+      <data-table :items="filteredPrices" :headers="headers" :loading="loading">
+        <template #item.fromAsset="{ item }">
+          <asset-details :asset="item.fromAsset" />
+        </template>
+        <template #item.toAsset="{ item }">
+          <asset-details :asset="item.toAsset" />
+        </template>
+        <template #item.price="{ item }">
+          <amount-display :value="item.price" />
+        </template>
+        <template #item.isWorth>
+          {{ tc('price_table.is_worth') }}
+        </template>
+        <template #item.usdPrice="{ item }">
+          <amount-display
+            v-if="item.usdPrice && item.usdPrice.gte(0)"
+            show-currency="symbol"
+            fiat-currency="USD"
+            tooltip
+            :value="item.usdPrice"
+          />
+          <div v-else class="d-flex justify-end">
+            <v-skeleton-loader width="70" type="text" />
+          </div>
+        </template>
+        <template #item.actions="{ item }">
+          <row-actions
+            :disabled="loading"
+            :delete-tooltip="tc('price_table.actions.delete.tooltip')"
+            :edit-tooltip="tc('price_table.actions.edit.tooltip')"
+            @delete-click="pending = item"
+            @edit-click="$emit('edit', item)"
+          />
+        </template>
+      </data-table>
+      <confirm-dialog
+        :title="tc('price_table.delete.dialog.title')"
+        :message="tc('price_table.delete.dialog.message')"
+        :display="showConfirmation"
+        @confirm="deletePrice"
+        @cancel="dismiss"
+      />
+    </card>
+  </div>
+</template>
+
+<script setup lang="ts">
+import { BigNumber } from '@rotki/common';
+import { NotificationPayload, Severity } from '@rotki/common/lib/messages';
+import { get, set } from '@vueuse/core';
+import { storeToRefs } from 'pinia';
+import { computed, ComputedRef, onMounted, PropType, ref, watch } from 'vue';
+import { useI18n } from 'vue-i18n-composable';
```

Comment:
nitpick: here the same except `ComputedRef` and `PropType` the other imports can be removed